noether_simulator/src/noether_simulator.cpp

```diff
@@ -50,15 +50,17 @@ namespace noether_simulator
     simulation_points_ = vtkSmartPointer<vtkPolyData>::New();
   }
 
+  //Integrate in x^2 and z^2
```

Comment:
This is too terse to remember what's going on. 
This is a definite integral of the function 10*(x^2 + z^2) from one point pt1 to another point pt2.